python/ray/tests/test_basic_2.py

```diff
@@ -60,9 +60,9 @@ def f2(x, y, *args):
     assert ray.get(f1.remote()) == ()
     assert ray.get(f1.remote(1)) == (1, )
     assert ray.get(f1.remote(1, 2, 3)) == (1, 2, 3)
-    with pytest.raises(Exception):
+    with pytest.raises(TypeError):
```

Comment:
Are these changes necessary for the test to work, or are they just making the tests more specific (better)

Reply:
Making the tests more specific for both client and driver modes.

Reply:
Fantastic :) This is definitely helpful!